.goreleaser.yaml

```
@@ -26,11 +27,11 @@ archives:
       {{- else }}{{ .Os }}{{ end }}_{{ .Arch }}
     format_overrides:
     - goos: windows
-      format: zip
+      formats: [ 'zip' ]
```

nit:
```suggestion
      formats: [ zip ]
```
Quotes are optional around strings

---

Right, I just went for the suggestion from the deprecation guide here.